SDK/ViewControllers/PageViewController.swift

```diff
@@ -308,8 +308,14 @@ final class PageViewController<PageQuery: Query>: UIViewController, UITableViewD
         
         // Show the seleced page in a PageViewController
         if pageSection.items[indexPath.row].itemClass == .page {
-            let pageVC = PageViewController<SinglePageQuery>(apiClient: apiClient,
-                                                             pageQuery: SinglePageQuery(pageID: String(pageSection.items[indexPath.row].itemID!)))
+            let languageSetting = SettingsManager.get(type: .language)
+            let singlePageQuery = SinglePageQuery(pageID: String(pageSection.items[indexPath.row].itemID!),
+                                                  locale: languageSetting.code)
+            let extractedExpr = PageViewController<SinglePageQuery>(apiClient: apiClient,
```

Comment:
Original comment by @dmfs: what's this redundant variable for? Why not assign to pageVC directly?
https://github.com/schedjoules/ios-public-calendars-sdk/pull/19#discussion_r245273186

Reply:
Removed the property in commit https://github.com/schedjoules/ios-public-calendars-sdk/pull/23/commits/748e0dca17d37ba8690fd500a779c6d71eab7a6b